### ISSUE TITLE
`customerId` can be `null`, but types state it's always a `number`

### ISSUE THREAD

**Issue by spaceemotion** [lmsqueezy/lemonsqueezy.js#84]
When using the subscriptions listing API, some subscriptions do not return a customer ID. While the name and email of the customer is present, the record can be missing. Even when opening the subscription via the dashboard, the customer cannot be clicked at.

```json5
{
	"billing_anchor": 23,
	"cancelled": true,
	"card_brand": null,
	"card_last_four": null,
	"created_at": "2024-02-23T15:49:22.000000Z",
	"customer_id": null, // <-- this number can be null
	"ends_at": "2024-03-04T16:52:04.000000Z",
	"first_subscription_item": {
		// ...
	},
	"order_id": 12345,
	"order_item_id": 12345,
	"pause": null,
	"product_id": 12345,
	"product_name": "Example",
	"renews_at": null,
	"status": "expired",
	"status_formatted": "Expired",
	"store_id": 12345,
	"test_mode": false,
	"trial_ends_at": null,
	"updated_at": "2024-03-21T12:30:26.000000Z",
	"urls": {
		// ...
	},
	"user_email": "test@example.com",
	"user_name": "John Doe",
	"variant_id": 12345,
	"variant_name": "Monthly"
}
```

**Comment by keyding** [comment]
Hi, @spaceemotion 

I've submitted the fix PR #85.

Also, could you please tell me how you subscribe to a subscription that will have a `customer_id` of `null`, I will update the test case. Thank you.

**Comment by brankoconjic** [comment]
Thank you for reporting this issue. It appears to be a bug on our side. We're currently investigating further and will provide updates as soon as we have more information.
